### ISSUE TITLE
Unify the work from the binder bod on PRs in a single repo, here maybe?

### ISSUE THREAD

**Issue by goanpeca** [jupyterlab/jupyterlab-bot#3]

**Comment by blink1073** [comment]
Here's @ian-r-rose's bot: https://github.com/ian-r-rose/jupyterlab-dev-mode-bot

**Comment by goanpeca** [comment]
@ian-r-rose is that working as a Github action?

**Comment by ian-r-rose** [comment]
No, it predates GitHub actions and uses webhooks. I would be in favor of transferring the functionality here and retiring that bot.

**Comment by goanpeca** [comment]
>No, it predates GitHub actions and uses webhooks

Ok, this is also using webhooks and running on Heroku free tier.

>I would be in favor of transferring the functionality here and retiring that bot.

I will look into it. Thanks!
